README.md

```diff
@@ -114,58 +114,58 @@ $ cargo run -- --host http://local.dev/
      Running `target/debug/loadtest --host 'http://local.dev/'`
 
 === PER TASK METRICS ===
------------------------------------------------------------------------------- 
+------------------------------------------------------------------------------
```

Comment:
All of these are cut & paste directly from Goose output -- as such, when this gets updated when something changes, these white spaces will return. As such, ideally `src/metrics.rs` should be updated to not generate this extra white-space in the first place.